draftjs_exporter/dom.py

```diff
@@ -28,7 +28,7 @@ class DOM(object):
     LXML = 'lxml'
     STRING = 'string'
 
-    dom = DOM_HTML5LIB
+    dom = import_string('draftjs_exporter.engines.html5lib.DOM_HTML5LIB')
```

Comment:
Not strictly necessary, but I did this for consistency's sake.

Reply:
I think we should remove this, otherwise it will load the `DOM_HTML5LIB` engine event if someone choose another engine.